apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -452,13 +554,86 @@ type ApplyStrategy struct {
 	// +kubebuilder:validation:Optional
 	ServerSideApplyConfig *ServerSideApplyConfig `json:"serverSideApplyConfig,omitempty"`
 
-	// TakeoverAction describes the action to take when we place the selected resources on the target cluster the first time.
+	// TakeoverAction determines the action to take when Fleet applies resources to a member
```

Comment:
Shall we change this to WhenToTakeOver (to be consistent with WhenToApply)?
WhenToTakeOver: Always, IfNoDiff.
WhenToApply: Always, IfNoDrift.

I assume the API hasn't been implemented.

Reply:
Done 👌